improver_tests/acceptance/SHA256SUMS

```diff
@@ -145,12 +145,13 @@ f04e1b0908405a54d83b2f05e8849a56e431282e539b139bc59b61889e91090b  ./generate-met
 1f886358f7392e356787ba3a7a6ecaa999e26764a9441041f8fb932c64590e6a  ./generate-metadata-cube/single_height_level.json
 4e6691746499e453b38e4accf08cbf3b6b69e47ca60b0e1c6e1d3dfacb4767f5  ./generate-metadata-cube/thresholds.json
 418d6365dd3d7acd8be77d4ce20d4b9a6a96bdd4767cdafc67c4971bf940ec63  ./generate-metadata-cube/variable_cube_all_inputs.json
-7778adb43f02e43b394ae9dd4309656fb34153aa890f5840077b45bcaf9f016d  ./generate-orographic-smoothing-coefficients/basic/kgo.nc
-79d407f646d3a18adc327557456b89c2c9677f865ca85960114bc873c1d8591a  ./generate-orographic-smoothing-coefficients/basic/kgo_different_limits.nc
-9b297e95777f3c6e47eb02377512488feb3ea6e8f193c049faea5a29ebbf5a65  ./generate-orographic-smoothing-coefficients/inverse_mask_zeroed/kgo.nc
+257fb45f2f35dcc8bce59d243df7c3f1dd38008880d1efec5f183ae474421dfb  ./generate-orographic-smoothing-coefficients/basic/kgo.nc
+f263d306a0cc498376084f1eb68a72ea4fdad46af08ff4e650fdb019af1f4edf  ./generate-orographic-smoothing-coefficients/basic/kgo_different_limits.nc
+465f9d75487fb57cee04aa8f815cdb59fded121b13936eac8c847832227eb558  ./generate-orographic-smoothing-coefficients/basic/kgo_different_power.nc
+1378c6dac85992ffb7cf92d428bc2c0479f99b230ded3388e8c3833842bb67b1  ./generate-orographic-smoothing-coefficients/inverse_mask_zeroed/kgo.nc
 6367be7ab2d52fea53f798b8857a1aa98b78d8cfa29fcea26c5fe7215644d602  ./generate-orographic-smoothing-coefficients/landmask.nc
-717f694b0556b029a05cff97e9fb14196b6dac6af1f6b39741a73057052b0d3e  ./generate-orographic-smoothing-coefficients/mask_boundary/kgo.nc
-ffda9ba4b0c59282d5f8b1131d9c6f46edf92697a187f0edf146f1112c637ee0  ./generate-orographic-smoothing-coefficients/mask_zeroed/kgo.nc
+f7fa3c78eb22222a33d3b688a4acb76533de53c32d73f45034d09b62d2d47052  ./generate-orographic-smoothing-coefficients/mask_boundary/kgo.nc
+8eeb91fd19f025aebf27327c1753178857187c3a4818bc7b47eb4a38d3dfaff4  ./generate-orographic-smoothing-coefficients/mask_zeroed/kgo.nc
```

Comment:
I don't think these outputs should have changed.  Could you please:

- Rebase your IMPROVER branch on master
- Rebase / merge your test data branch
- Regenerate checksums again and check that only the expected files have been updated

Reply:
I will make sure I do these things, as you are right that only the threshold checksums should have changed.